src/borg/archiver.py

```diff
@@ -2234,10 +2234,10 @@ def do_break_lock(self, args, repository):
         /path/to/repo /home/user`` will store all files as
         `home/user/.../file.ext`.
 
-        A folder exclusion pattern can end either with or without a slash ('/').
-        If it ends with a slash, such as `some/path/`, the directory will be
-        included but not its content. If it does not end with a slash, such as
-        `some/path`, both the directory and content will be excluded.
+        A directory exclusion pattern can end either with or without a slash
+        ('/'). If it ends with a slash, such as `some/path/`, the directory will
```

Comment:
the upper line is only 68 chars long, so it would look better (and also not cause issues) if you kept the end of the sentence on same line.

Reply:
(for html, it of course makes no difference, but it does when read on the terminal, like `borg help patterns`)

Reply:
Sure, I've updated it now.